docs/design-language.md

```diff
@@ -105,28 +99,32 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam nec commodo eli
 ### List Controls
 
 ![list](/img/list/list-controls.png "List Controls")
-![list](/img/list/list-controls-checkbox.png "List Controls (checkbox)")
-![list](/img/list/list-controls-toggle.png "List Controls (toggle)")
 
 ### Pinned Subheader List
 
 ## ![list](/img/list/pinned-subheader.png "Pinned Subheader List")
 
 ## States
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam nec commodo elit. Curabitur dui nulla, vulputate id rutrum sed, maximus consequat eros. Nullam pharetra urna quis sapien tincidunt, et iaculis mauris ornare. Vestibulum convallis et felis at lobortis. Nulla bibendum vel quam quis accumsan.
+A few guidelines should be followed to ensure that the list element looks uniform, and fit within the application properly.
 
 ### Hover
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam nec commodo elit. Curabitur dui nulla, vulputate id rutrum sed, maximus consequat eros. Nullam pharetra urna quis sapien tincidunt, et iaculis mauris ornare. Vestibulum convallis et felis at lobortis. Nulla bibendum vel quam quis accumsan.
+![list](/img/list/selected-item.png "Hover Item State")
+
+When a hover state is triggered, it cannot just rely on a color change. There also must be a state change that shows the differentiation. For example, in our programs, there is a background color change, as well as the addition of a side border color. Hover states always include a styling change, and the background color will get a gradient.
```

Comment:
Can I suggest adding in some wording to explain why we can't just use color when hovering? Something along the lines of, "A hover state cannot rely on a color change as the only indication that it is being hovered due to users that may be colorblind or have other visual impairments. In our applications we will typically add a border along with a color change to be WCAG compliant."